dev_guide/app_tutorials/ruby_on_rails.adoc

```diff
@@ -0,0 +1,357 @@
+= Ruby on Rails
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Ruby on Rails is a popular web framework written in https://github.com/openshift/mysql/tree/master/5.5[Ruby].
+This guide covers using Rails 4 on OpenShift.
+
+[WARNING]
+====
+We strongly advise going through the whole tutorial to have an overview of all the steps necessary to run
+your application on the OpenShift. If you experience a problem try reading through the entire tutorial and then going back to your issue. It can also be useful to review your previous steps to ensure that all the steps were executed correctly.
+====
+
+For this guide you will need:
+
+* Basic Ruby/Rails knowledge
+* Locally installed version of Ruby 2.0.0+, Rubygems, Bundler
+* Basic Git knowledge
+* Running instance of OpenShift v3
+
+== Local Workstation Setup
+
+First make sure that an instance of OpenShift is running and is available. For more info on
+how the get OpenShift up and running check the link:../../getting_started/administrators.html#installation-methods[installation methods]. Also make sure that your link:../../cli_reference/get_started_cli.html[`oc` CLI client is installed] and the command is accessible from your command shell, so you can use it to link:../../cli_reference/get_started_cli.html#basic-setup-and-login[log in] using your email address and password.
```

Comment:
s/how the/how to/

May be better to link to `../../install_config/install/index.html` for "installation methods", in the interest of making it flow well for both OSE and Origin.